playgrounds/vite/src/App.tsx

```diff
@@ -1,11 +1,53 @@
-import { Button } from '@itwin/itwinui-react';
+import { SvgAirplane } from '@itwin/itwinui-icons-react';
+import { Button, FileUpload, FileUploadCard } from '@itwin/itwinui-react';
+import React from 'react';
 
 const App = () => {
+  const [files, setFiles] = React.useState<Array<File>>([]);
+
   return (
     <>
-      <Button>Hello world</Button>
+      <FileUpload onFileDropped={() => console.log('Files were dropped')}>
+        <FileUploadCard />
+      </FileUpload>
+      <FileUpload onFileDropped={() => console.log('Files were dropped')}>
+        <FileUploadCard
+          onFileChange={(e) => {
+            setFiles(Array.from(e.target.files || []));
+          }}
+        >
+          {<SvgAirplane></SvgAirplane>}
+          <FileUploadCard.Text>
+            <FileUploadCard.Label>
+              {files.length ? files[0].name : undefined}
+            </FileUploadCard.Label>
+            <FileUploadCard.Description>
+              {files.length ? files[0].name : undefined}
+            </FileUploadCard.Description>
+          </FileUploadCard.Text>
+          <FileUploadCard.Action>
+            <Button onClick={(e) => setFiles([])}>Clear</Button>
+          </FileUploadCard.Action>
+        </FileUploadCard>
+      </FileUpload>
     </>
   );
 };
 
+// const WithDiscrd = () => {
```

Comment:
all changes in App.tsx should be discard instead of being pushed

Reply:
This has been removed